packages/maskbook/src/plugins/ITO/assets/tokenIcon.tsx

```diff
@@ -0,0 +1,67 @@
+export const EthIcon = ({ size = 24 }: { size?: number }) => (
+    <svg width={size} height={size} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please use https://material-ui.com/api/svg-icon/#svgicon-api to wrap your icons

Reply:
In this case, `<TokenIcon />` is enough.